Parser/workhorseParse.m

```diff
@@ -167,13 +168,42 @@
   
   %
   % calculate velocity (speed and direction)
-  % currently assuming earth coordinate transform
+  % check for ENU and Beam coordinates. Instrument coordinates not
+  % considered.
   %
-  
-  veast = velocity.velocity1;
-  vnrth = velocity.velocity2;
-  wvel  = velocity.velocity3;
-  evel  = velocity.velocity4;
+  adcpOrientations = str2num(fixed.systemConfiguration(:, 1)); % str2num is actually more relevant than str2double here
+  adcpOrientation = mode(adcpOrientations); % hopefully the most frequent value reflects the orientation when deployed
```

Comment:
this value should be exported forward in meta.

Reply:
Yes, agree. 
Also, there is a bit of an issue with using the mode and hoping it is correct. If the record is very short (due to a failure), and the majority of the data is 'out of water' and facing the other way, then the orientation will be identified incorrectly. There is no way for the user to put in their own selection to override the automatic choice. Maybe this is a new issue.